### ISSUE TITLE
Media type

### ISSUE THREAD

**Issue by martinthomson** [geojson/draft-geojson#114]
The current media type uses a `vnd.`-prefixed name.  If this cannot change, then the document should explain why this is the case (backward compatibility no doubt), and probably leave another coin at the altar of RFC 6648.

Either way, we should start the review process.  I don't see the value in the IANA registry.

**Comment by dret** [comment]
On 2015-11-17 01:09, Martin Thomson wrote:
> Either way, we should start the review process. I don't see the value in
> the IANA registry.

the current draft has an IANA section that will prompt the media type to 
be registered, right? isn't that good enough for now until we're done 
and the registry entry will be created?

**Comment by sgillies** [comment]
@martinthomson when you say you don't see the value in the IANA registry, do you mean that you're in favor of registering `application/geo+json` with the IETF?

**Comment by martinthomson** [comment]
@dret, the review process for the registry can slow things down if we don't start now.

@sgillies, a registration is a fine idea, but not with `vnd.`, that's all.

**Comment by sgillies** [comment]
@martinthomson I've got a change to the IANA considerations section in #124. What's my next action, submission to http://www.iana.org/form/media-types?

**Comment by martinthomson** [comment]
I think that IANA usually picks up these sorts of things during the document processing phases.  However, I think that you should send an email for preliminary review here: media-types@iana.org -- see http://tools.ietf.org/html/rfc6838#section-5.1

**Comment by sgillies** [comment]
Done.

**Comment by martinthomson** [comment]
@sgillies, are we free to close this?  It looks like we can leave the other steps to the RFC editor/IANA for when publication is happening.

**Comment by martinthomson** [comment]
Ahh, I see that there are a few changes from the discussion on the media-types list:

> I suggest marking the application/vnd.geo+json registration *deprecated* in
the docs.

> Maybe ask IANA to add a note on the vnd. version.

> >  Windows clipboard name: GeoJSON
>>  Macintosh uniform type identifier:  public.geojson conforms to public.json
>
> Such an addition would allow applications that consider exchanging
> GeoJSON content to do so by employing such gestures as copy-and-paste or
> drag-and-drop. These fields are not (yet) in the process but has been
> suggested.

> I also believe the line "Macintosh file type code : TEXT" should be
> replaced by the value n/a as this identifier system is not in use
> anymore (and TEXT is a bit too general).

**Comment by dret** [comment]
depending on how #97 plays out, maybe the media type registration should include an optional "profile" media type parameter (as described in https://tools.ietf.org/html/rfc6906#section-3.1).

**Comment by sgillies** [comment]
I've got a IANA change request ready to go, but have paused to ask: before requesting that application/vnd.geo+json be marked obsolete should we not get the new draft-ietf-geojson-01 (with the application/geo+json definition) published?

**Comment by martinthomson** [comment]
Yes, and frankly, I'm not sure that it's worth asking to mark the old one obsoleted.  The media type registrations page doesn't really have any provision for that.  The best we could do is point the old media type at this document, but then we don't have a good definition for it.  It's not clear that creating that definition is a good idea.  I'll open an issue.

**Comment by dret** [comment]
On 2016-02-02 20:31, Sean Gillies wrote:
> I've got a IANA change request ready to go, but have paused to ask:
> before requesting that application/vnd.geo+json be marked obsolete
> should we not get the new draft-ietf-geojson-01 (with the
> application/geo+json definition) published?

also, can we wait on how #97 plays out and see if an optional "profile" 
parameter gets added?

**Comment by sgillies** [comment]
@dret definitely.

**Comment by dret** [comment]
On 2016-02-02 20:43, Martin Thomson wrote:
> Yes, and frankly, I'm not sure that it's worth asking to mark the old
> one obsoleted. The media type registrations page doesn't really have any
> provision for that. The best we could do is point the old media type at
> this document, but then we don't have a good definition for it. It's not
> clear that creating that definition is a good idea. I'll open an issue.

there's #125 proposing to summarize the changes between the old and the 
new spec. that could be a pretty good reference for explaining how the 
old and new media types relate.

**Comment by sgillies** [comment]
A helpful response from Tony Hansen:

> Sean,

> I like the idea of documenting the differences from application/vnd.geo+json and using an IANA note saying that application/vnd.geo+json should be considered obsolete.

> This should be sufficient to make the change occur in the applicatino/vnd.geo+json. Words along these lines should be fine:

> IANA Considerations

> The registration entry for applciation/vnd.geo+json in <http://www.iana.org/assignments/media-types/> should have its status changed to be Obsolete with a pointer to the media type application/geo+json and a reference added to this RFC.

> IANA would then change the entry there to say
application/vnd.geo+json – OBSOLETED in favor of application/geo+json

> If IANA wants the wording of this changed, they’ll let you know when they review your document prior to publication as an RFC.

> As the change owner of record for application/vnd.geo+json, you certainly may also fill in the form at <http://www.iana.org/form/media-types> and provide the updated status as you suggested, but it shouldn’t be necessary. (Note that if you had NOT been writing an RFC about the replacement media type, then using the online form would definitely have been the proper course of action.)

> Tony Hansen

I'm going to make another PR to do this.
